src/ripple/app/main/Main.cpp

```diff
@@ -394,6 +395,11 @@ int run (int argc, char** argv)
         try
         {
             config->VALIDATION_QUORUM = vm["quorum"].as <std::size_t> ();
+            if (vm["quorum"].as <std::size_t>() < 1)
```

Comment:
Just use `if(config->VALIDATION_QUORUM)`  instead. Makes it clearer what's happening.